packages/client/src/store/actions/bookingOperations.ts

```diff
@@ -130,6 +134,50 @@ export const updateBookingNotes: UpdateBooking<{ bookingNotes: string }> =
     }
   };
 
+/**
+ * Updates customer data in bookings collection
+ * @param payload.secretKey {string} - customer secretKey
+ * @param payload.customer {Customer} - cutomer type
+ * @returns FirestoreThunk
+ */
+export const updateBookingCustomer: {
+  (paylod: {
+    secretKey: Customer["secretKey"];
+    customer: Customer;
+  }): FirestoreThunk;
+} =
+  ({ secretKey, customer }) =>
+  async (dispatch, getState) => {
+    const organization = getOrganization();
+
+    try {
+      const db = getFirestore();
```

Comment:
When the form is submitted we want to invoke a cloud function. See third point "[On form submit a (to be implemented) cloud function should be invoked](https://github.com/eisbuk/EisBuk/issues/667)" on #667.

The `updateBookingCustomer` action should be changed to invoke a (new, to be written) cloud function, instead of mutating firestore entries from the client.